### ISSUE TITLE
Honeybadger testing of AWS v20 together with Vintage-CAPA migration

### ISSUE THREAD

**Issue by T-Kukawka** [giantswarm/roadmap#3210]
The time has come to start testing final releases as well as migration from Vintage `v20` to CAPA. We have created a dedicated Vintage MC `garfish` to perform any vintage or migration testing for stability purposes. The dedicated CAPA cluster for migration will be the CAPA stable-testing MC `grizzly`.

We would kindly ask all teams to perform comprehensive tests for 3 use-cases, ordered in terms of priorities if they can't be performed all at once.

### 1. Vintage [`AWS v20`](https://github.com/giantswarm/releases/pull/1204) 
***Cluster creation on `garfish` - `giantswarm` Organization***

This is the last release of Vintage containing `1.25 k8s`. The `1.25` kubernetes introduces a breaking change in terms of removal `PSPs` from its API, meaning that all workloads will have to comply with the `global` toggle disabling` PSPs` as in `19.3.x` release. Prior to making `v20` release available to customers, we need to validate that all applications are running smoothly. The Vintage tests are standard as always - you just create the `v20` release and validate your applications. Separate stable MC in this case will guarantee no manual changes in the release and stability.

- [x] Honeybadger finished with Vintage `v20` testing (please mark it in main issue as well)

### 2. CAPA [`0.60.0`](https://github.com/giantswarm/cluster-aws/blob/master/CHANGELOG.md#0600---2024-01-29) 
***Cluster creation on `grizzly` - `giantswarm` Organization - be aware that this is production MC, so it will page everyone. In practice any CAPA MC should work for this test.***

Starting with `cluster-aws`-`v0.60.0` and `default-apps-aws`-`v0.45.1` onwards CAPA supports Kubernetes 1.25 with all needed features to run our workloads in the same manner as on VIntage clusters. Please for testing use always latest [`cluster-aws`](https://github.com/giantswarm/cluster-aws/blob/master/CHANGELOG.md) as well as [`default-apps-aws`](https://github.com/giantswarm/default-apps-aws/blob/master/CHANGELOG.md) releases.

- [x] Honeybadger finished with CAPA 1.25 testing (please mark it in main issue as well)

### 3. Vintage to CAPA migration 
***Cluster creation for migration on `garfish` - `capa-migration-testing` Organization. Clusters will be migrated to `grizzly` - `capa-migration-testing` Organization.***

Phoenix and Honeybadger worked extensively on making the migration as smooth as possible. The [`migration-cli`](https://github.com/giantswarm/capi-migration-cli) has been introduced that orchestrates migration of apps as well as infrastructure. Here the main point is to discover if your application and any custom configurations that could be applied by customers are migrated properly.

The `migration-cli` has been extended to facilitate easy testing for all teams ad Giant Swarm. Please follow the requirements as well as the procedure that is described in the [`tests` section](https://github.com/giantswarm/capi-migration-cli/tree/main/tests) of the tool. In case of any issue with infrastructure - ping Phoenix, if the app/configmap migration will face any issues or inconsistencies - ping Honeybadger.

- [x] Honeybadger finished with Vintage to CAPA migration testing (please mark it in main issue as well)

```[tasklist]
### Tasks
```

**Comment by T-Kukawka** [comment]
I have made adjustments in the tracking ticket as well as the teams tickets regarding the CAPA and migration testing instructions.

TL;DR: Testing of CAPA/Migration is moved from `gazelle` to `grizzly`

Initially `gazelle` has been chosen to test the CAPA migration as it is a Production MC, meaning most stable one. However this has resulted in unforeseen pages towards kaas-cloud oncall that we would like to limit.

We do recognise the pages and also actively work on testing, hence such pages are just a distraction away from the operations clusters that most of the teams have migrated the GS production workloads on.

Taking all the facts into consideration we have decided that it would be best to move the testing to `grizzly` which is stable-testing installation. Installation is primarily running e2e test and is treated as stable (no changes on the MCs).
Thanks for understanding and let us know if something is not working

**Comment by nce** [comment]
Finally migration was an overall success.

All apps are migrated and running on/from the capi mc. Some minor issues found on the helm labels, which i'll think about.


```
(N/A) ~/v/g/g/c/tests ❯❯❯ ./tests -cluster-name ulli10 -cluster-organization ulli -mc-capi grizzly                             
Executing opsctl login garfish, check your browser for dex login

Logging in to the management cluster of installation garfish

Executing the command

  kubectl gs login https://g8s.garfish.eu-central-1.aws.gigantic.io --cluster-admin


Your browser should now be opening this URL:
https://dex.g8s.garfish.eu-central-1.aws.gigantic.io/auth?access_type=offline&client_id=zQiFLUnrTFQwrybYzeY53hWWfhOKWRAU&redirect_uri=http%3A%2F%2Flocalhost%3A59667%2Foauth%2Fcallback&response_type=code&scope=openid+profile+email+groups+offline_access+audience%3Aserver%3Aclient_id%3Adex-k8s-authenticator&state=B7FND6aNnhhrllUlqLEwfXeLzleLhG0OJJSDSlVvFOM%3D&connector_filter=giantswarm

Logged in successfully as 'ulli' on cluster 'garfish'.

A new kubectl context 'gs-garfish' has been created and selected. To switch back to this context later, use either of these commands:

  kubectl gs login garfish
  kubectl config use-context gs-garfish
Executing kubectl gs template cluster --provider=aws --organization=ulli --name=ulli10 --release=v20.0.0 --description=Vintage-to-CAPI CLI migration test cluster

Executing kubectl gs template nodepool --provider=aws --organization=ulli --release=v20.0.0 --cluster-name=ulli10 --description=Vintage-to-CAPI CLI migration node pool cluster --availability-zones=eu-central-1a,eu-central-1b,eu-central-1c

Executing kubectl --context=gs-garfish apply -f ulli10-cluster-vintage.yaml

cluster.cluster.x-k8s.io/ulli10 created
awscluster.infrastructure.giantswarm.io/ulli10 created
g8scontrolplane.infrastructure.giantswarm.io/jspu4z8568 created
awscontrolplane.infrastructure.giantswarm.io/jspu4z8568 created
machinedeployment.cluster.x-k8s.io/vu93au66da created
awsmachinedeployment.infrastructure.giantswarm.io/vu93au66da created
Vintage cluster manifest applied successfully

Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is not yet in Created state. Retrying in 5 minutes.
Vintage cluster ulli10 is in Created state
Vintage nodepool vu93au66da is ready
All nodepools are ready
Security bundle is deployed

Vintage cluster ulli10 is ready for migration

=======================================
now its time to do any changes you want to do before the migration to CAPI starts.
Please type 'Yes' to confirm to proceed further:
> Yes
Confirmed! Continuing.

Checking kubernetes client for Vintage MC garfish.
Connected to gs-garfish, k8s server version v1.24.17
Checking kubernetes client for the cluster ulli10.
Context for cluster [garfish ulli10] not found, executing 'opsctl login', check your browser window.
Connected to gs-garfish-ulli10-clientcert, k8s server version v1.25.16
Checking kubernetes client for CAPI MC grizzly.
Context for cluster [grizzly] not found, executing 'opsctl login', check your browser window.
Connected to gs-grizzly, k8s server version v1.25.16
Generating AWS credentials for cluster garfish/ulli10.
Assume Role MFA token code: 198861
Generated AWS credentials for arn:aws:sts::381491998795:assumed-role/GiantSwarmAdmin/ulli@giantswarm.io-opsctl-cli
Checking garfish's vault connection
Connected to vault 1.7.10
Init phase finished.

Starting migration of cluster ulli10 from garfish to grizzly

Preparation Phase
Fetching vintage CRs
Deleting etcd-kubernetes-resources-count-exporter vintage app.
Deleted vintage app etcd-kubernetes-resources-count-exporter
Deleting chart-operator-extensions vintage app.
Deleted vintage app chart-operator-extensions
Migrating secrets to CAPI MC
Migrated ulli10-cilium-user-values-extraconfig configmap
Migrated ulli10-observability-bundle-logging-extraconfig configmap
Successfully migrated cluster AWS default apps values.
Successfully migrated cluster IAM account role to AWSClusterRoleIdentity.
Successfully disabled vintage machine health check.

Executing the following command to migrate non-default apps via external tool:
app-migration-cli prepare -s garfish -d grizzly -n ulli10 -o org-ulli
Connected to gs-garfish, k8s server version v1.24.17
Connected to gs-grizzly, k8s server version v1.25.16
Apps (20) and config is dumped and migrated to disk: /Users/nce/vcs/github/giantswarm/capi-migration-cli/tests/garfish-ulli10-apps.yaml
Successfully prepared non-default apps migration.
Successfully scaled down Vintage app operator deployment.
Cleaned legacy chart cilium
Cleaned legacy chart aws-ebs-csi-driver
Cleaned legacy chart aws-cloud-controller-manager
Cleaned legacy chart coredns
Cleaned legacy chart vertical-pod-autoscaler-crd
Preparation phase completed.

Skipping stopping reconciliation of CRs on vintage cluster
Generated CAPI cluster template manifest for cluster ulli10
Generated CAPI default-apps template manifest for cluster ulli10
Applying CAPI cluster APP CR to MC
CAPI Cluster app applied successfully.

Waiting for 1 CAPI node with status Ready
..............Registering instance i-0fdb40cab8a734874 with ELB ulli10-api
Registering instance i-0fdb40cab8a734874 with ELB ulli10-api-internal
...................
CAPI node ip-10-1-2-47.eu-central-1.compute.internal ready. 1/1

Found CAPI 1 nodes with status Ready, waited for 330 sec.
Removed initial-cluster from kubeadm configmap
Removed initial-cluster from kubeadm configmap
Applying CAPI cluster APP CR to MC
CAPI Cluster app applied successfully.

Removing static manifests for control-plane components on all vintage nodes.
{"level":"info","ts":"2024-03-13T15:13:10+01:00","logger":"KubeAPIWarningLogger","msg":"kyverno.io/v2alpha1 PolicyException is deprecated; use kyverno.io/v2beta1 PolicyException"}
Applied PolicyExceptions for CP node cleanup job
Created job disable-cp-ip-10-1-2-102.eu-central-1.compute.internal
Created job disable-cp-ip-10-1-2-168.eu-central-1.compute.internal
Created job disable-cp-ip-10-1-2-44.eu-central-1.compute.internal
Waiting until all jobs finished
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal not finished yet, waiting 5 sec.
Job 1/3 - disable-cp-ip-10-1-2-102.eu-central-1.compute.internal finished.
Job 2/3 - disable-cp-ip-10-1-2-168.eu-central-1.compute.internal finished.
Job 3/3 - disable-cp-ip-10-1-2-44.eu-central-1.compute.internal finished.

Deleted PolicyExceptions for CP node cleanup job
Cordon all vintage control plane nodes.
Found 3 nodes for cordoning
Deleted pod org-ulli/ulli10-app-operator-9f99cc8f-m2rs7 on CAPI MC to force reconcilation
Deleted pod giantswarm/chart-operator-84d59fc589-tz48d to reschedule it on CAPI control plane node
Deleted pod kube-system/cilium-6qq2l
Deleted pod kube-system/cilium-l6sfm
Deleted pod kube-system/cilium-lbg6q
Deleted pod kube-system/cilium-lzfsg
Deleted pod kube-system/cilium-operator-58d97b7594-2fbtk
Deleted pod kube-system/cilium-operator-58d97b7594-fqjw5
Deleted pod kube-system/cilium-tdg2v
Deleted pod kube-system/cilium-wcw2q
Deleted pod kube-system/cilium-wv5qj
Deleted pod kube-system/hubble-relay-588c58df8f-746t2
Deleted pod kube-system/hubble-ui-c7577d476-bj9kq
Waiting for 3 CAPI node with status Ready

CAPI node ip-10-1-2-47.eu-central-1.compute.internal ready. 1/3
.Deleted crashed pod kube-system/hubble-relay-588c58df8f-746t2
Deleted crashed pod kube-system/hubble-relay-588c58df8f-746t2
.Deleted crashed pod kube-system/hubble-relay-588c58df8f-746t2
Deleted crashed pod kube-system/hubble-relay-588c58df8f-746t2
.Deleted crashed pod kube-system/hubble-relay-588c58df8f-746t2
....Registering instance i-0f4a2303b1748b86b with ELB ulli10-api
Registering instance i-0f4a2303b1748b86b with ELB ulli10-api-internal
..Deleted crashed pod kube-system/cilium-operator-58d97b7594-hj824
...Registering instance i-09f4ba1d4d2052158 with ELB ulli10-api
Registering instance i-09f4ba1d4d2052158 with ELB ulli10-api-internal

CAPI node ip-10-1-2-190.eu-central-1.compute.internal ready. 2/3
.........
CAPI node ip-10-1-2-120.eu-central-1.compute.internal ready. 3/3

Found CAPI 3 nodes with status Ready, waited for 210 sec.
Draining all vintage control plane nodes.
Found 3 nodes for draining
Started draining node ip-10-1-2-102.eu-central-1.compute.internal
Started draining node ip-10-1-2-168.eu-central-1.compute.internal
Started draining node ip-10-1-2-44.eu-central-1.compute.internal
WARNING: ignoring DaemonSet-managed Pods: kube-system/aws-cloud-controller-manager-m9ptn, kube-system/capi-node-labeler-hqxjt, kube-system/cert-exporter-daemonset-w68hp, kube-system/cilium-bglzm, kube-system/k8s-audit-metrics-7tqgr, kube-system/k8s-dns-node-cache-8mhh9, kube-system/net-exporter-s77m9, kube-system/node-exporter-node-exporter-25drv, kube-system/prometheus-blackbox-exporter-22pht, kube-system/promtail-bpvl6
evicting pod kube-system/coredns-controlplane-84bd7c7655-xmf6b
evicting pod kube-system/disable-cp-ip-10-1-2-44.eu-central-1.compute.internal-6fg8t
WARNING: ignoring DaemonSet-managed Pods: kube-system/aws-cloud-controller-manager-f2xqw, kube-system/capi-node-labeler-tqhj5, kube-system/cert-exporter-daemonset-mp42r, kube-system/cilium-2697k, kube-system/k8s-audit-metrics-zwbgn, kube-system/k8s-dns-node-cache-jwj79, kube-system/net-exporter-5z7j2, kube-system/node-exporter-node-exporter-glgmx, kube-system/prometheus-blackbox-exporter-dxxvn, kube-system/promtail-ls5ks
evicting pod kube-system/disable-cp-ip-10-1-2-102.eu-central-1.compute.internal-7psl5
Finished draining node ip-10-1-2-102.eu-central-1.compute.internal
WARNING: ignoring DaemonSet-managed Pods: kube-system/aws-cloud-controller-manager-98kw2, kube-system/capi-node-labeler-lqcdd, kube-system/cert-exporter-daemonset-s96cl, kube-system/cilium-lshl7, kube-system/k8s-audit-metrics-klfxf, kube-system/k8s-dns-node-cache-nj4zj, kube-system/net-exporter-f7csq, kube-system/node-exporter-node-exporter-dkkjf, kube-system/prometheus-blackbox-exporter-7s6qf, kube-system/promtail-llqg9
evicting pod kube-system/disable-cp-ip-10-1-2-168.eu-central-1.compute.internal-wwq9n
evicting pod kube-system/cilium-operator-58d97b7594-nw9w9
Finished draining node ip-10-1-2-168.eu-central-1.compute.internal
Finished draining node ip-10-1-2-44.eu-central-1.compute.internal
Deleting vintage control plane ASGs.
AWS Credentials are still valid, no need to refresh
Found 3 ASG groups for deletion
Deleted ASG group cluster-ulli10-tccpn-ControlPlaneNodeAutoScalingGroup-DUUz7xHcWpaY
Terminated 1 instances in ASG group cluster-ulli10-tccpn-ControlPlaneNodeAutoScalingGroup-DUUz7xHcWpaY
Deleted ASG group cluster-ulli10-tccpn-ControlPlaneNodeAutoScalingGroup2-izpkaZILqsYk
Terminated 2 instances in ASG group cluster-ulli10-tccpn-ControlPlaneNodeAutoScalingGroup2-izpkaZILqsYk
Deleted ASG group cluster-ulli10-tccpn-ControlPlaneNodeAutoScalingGroup3-7tthKcy42hA6
Terminated 1 instances in ASG group cluster-ulli10-tccpn-ControlPlaneNodeAutoScalingGroup3-7tthKcy42hA6
Deleted vintage control plane ASGs.
Waiting KubeadmControlPlane to stabilise  (all replicas to be up to date and ready).
...........
KubeadmControlPlane is stabilised.
Waiting for all CAPI nodes in node pool vu93au66da to be ready.
Waiting for 3 CAPI node with status Ready

CAPI node ip-10-1-3-123.eu-central-1.compute.internal ready. 1/3

CAPI node ip-10-1-3-161.eu-central-1.compute.internal ready. 2/3

CAPI node ip-10-1-3-28.eu-central-1.compute.internal ready. 3/3

Found CAPI 3 nodes with status Ready, waited for 0 sec.
Draining all vintage worker nodes for nodepool vu93au66da.
Found 3 nodes for draining
Started draining node ip-10-1-3-15.eu-central-1.compute.internal
Started draining node ip-10-1-3-159.eu-central-1.compute.internal
Started draining node ip-10-1-3-82.eu-central-1.compute.internal
WARNING: ignoring DaemonSet-managed Pods: kube-system/capi-node-labeler-25pg5, kube-system/cert-exporter-daemonset-xptjc, kube-system/cilium-hdhvm, kube-system/ebs-csi-node-8fc8d, kube-system/grafana-agent-gjj9l, kube-system/k8s-dns-node-cache-g4v9k, kube-system/net-exporter-j4vbv, kube-system/node-exporter-node-exporter-k5bw4, kube-system/prometheus-blackbox-exporter-vllf9, kube-system/promtail-z6jkb
evicting pod kube-system/aws-pod-identity-webhook-app-74bd48dc77-srzt7
evicting pod ulli10/cm-acme-http-solver-k7cjk
evicting pod ulli10/cm-acme-http-solver-hc5hc
evicting pod ulli10/gitops-server-5c877454c7-6hghc
evicting pod kube-system/coredns-workers-95fcb8cc9-wbgmg
evicting pod kube-system/metrics-server-676d54c976-g7ns6
evicting pod giantswarm/app-admission-controller-769d7f7cc8-jjbrl
evicting pod kube-system/vertical-pod-autoscaler-admission-controller-7595f8c98b-4gqg5
evicting pod kube-system/vertical-pod-autoscaler-recommender-5f9f46b978-jc2sh
evicting pod kube-system/prometheus-prometheus-agent-0
evicting pod ulli10/app-exporter-968b48fd6-9lg2g
evicting pod ulli10/docs-proxy-app-5699d95bdd-r6sd4
evicting pod ulli10/config-controller-68d97764-4lqmd
evicting pod ulli10/crsync-6c45c5f67-skjmk
evicting pod ulli10/docs-indexer-app-intranet-28505650-fln6v
evicting pod ulli10/giantswarmio-nginx-app-875bcd944-sm2d8
evicting pod ulli10/external-secrets-webhook-9498c4546-jhln4
error when evicting pods/"docs-proxy-app-5699d95bdd-r6sd4" -n "ulli10" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
error when evicting pods/"app-admission-controller-769d7f7cc8-jjbrl" -n "giantswarm" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
WARNING: ignoring DaemonSet-managed Pods: kube-system/capi-node-labeler-tqq5j, kube-system/cert-exporter-daemonset-zz6bf, kube-system/cilium-hpzkr, kube-system/ebs-csi-node-2dmlj, kube-system/grafana-agent-ds5h5, kube-system/k8s-dns-node-cache-hhb6m, kube-system/net-exporter-kt4hp, kube-system/node-exporter-node-exporter-jvwqs, kube-system/prometheus-blackbox-exporter-zc2kc, kube-system/promtail-zr9df
evicting pod giantswarm/app-admission-controller-769d7f7cc8-5dwrb
evicting pod kube-system/aws-pod-identity-webhook-restarter-28505655-wl88z
evicting pod kube-system/external-dns-bd7c955cb-lkppg
evicting pod ulli10/schema-server-7bb476b45c-8m5jv
evicting pod kube-system/vertical-pod-autoscaler-admission-controller-7595f8c98b-k4d2c
evicting pod kube-system/cert-exporter-deployment-59974ff886-jq5h9
evicting pod kube-system/hubble-ui-c7577d476-smthw
evicting pod kyverno/kyverno-cleanup-cluster-admission-reports-28505650-7zxzj
evicting pod kube-system/cert-manager-app-cainjector-68f658bfbf-txv96
evicting pod kube-system/kube-prometheus-stack-kube-state-metrics-59d6fc7db-n6gts
evicting pod ulli10/cm-acme-http-solver-2fz9l
evicting pod kyverno/kyverno-cleanup-admission-reports-28505650-vttw7
evicting pod kube-system/aws-pod-identity-webhook-app-74bd48dc77-zknkh
evicting pod kube-system/hubble-relay-588c58df8f-f9bh5
evicting pod ulli10/backstage-bc5db5849-42xgt
evicting pod ulli10/blog-app-848c74cbf-46gz7
evicting pod ulli10/docs-indexer-app-docs-28505655-c7spd
evicting pod ulli10/cm-acme-http-solver-zc9bp
evicting pod kube-system/teleport-kube-agent-0
evicting pod ulli10/external-secrets-69c44bd777-kht8h
evicting pod ulli10/giantswarmio-nginx-app-875bcd944-rzm86
evicting pod ulli10/cm-acme-http-solver-26z9d
WARNING: ignoring DaemonSet-managed Pods: kube-system/capi-node-labeler-wq6mm, kube-system/cert-exporter-daemonset-s7x44, kube-system/cilium-5pt6q, kube-system/ebs-csi-node-6qjlb, kube-system/grafana-agent-94zfc, kube-system/k8s-dns-node-cache-bjfpj, kube-system/net-exporter-29p6m, kube-system/node-exporter-node-exporter-tzq8s, kube-system/prometheus-blackbox-exporter-jjwkz, kube-system/promtail-c6clw
evicting pod giantswarm/app-admission-controller-769d7f7cc8-fr678
evicting pod kube-system/apptest-app-588c5448bb-msdjz
evicting pod kube-system/vertical-pod-autoscaler-updater-db9449b85-9lg4b
evicting pod ulli10/docs-proxy-app-5699d95bdd-m8w85
evicting pod ulli10/schema-server-7bb476b45c-bg85w
evicting pod ulli10/blog-app-848c74cbf-4729h
evicting pod ulli10/hello-world-77dd5d5f7d-hp7wq
evicting pod kube-system/kube-prometheus-stack-operator-8b4d8d799-fq5tt
evicting pod security-bundle/kyverno-policy-operator-65dcf8f8f-46h5n
evicting pod ulli10/external-secrets-cert-controller-6c4d59b7bc-qv9hk
evicting pod ulli10/cm-acme-http-solver-cgxmf
evicting pod kube-system/coredns-workers-95fcb8cc9-592jn
evicting pod ulli10/cm-acme-http-solver-slwl2
evicting pod kube-system/teleport-kube-agent-1
evicting pod kube-system/aws-pod-identity-webhook-app-74bd48dc77-qpbbg
evicting pod ulli10/cm-acme-http-solver-w2tzm
evicting pod kube-system/metrics-server-676d54c976-cmctg

Finished draining node ip-10-1-3-82.eu-central-1.compute.internal
Deleting vintage vu93au66da node pool ASG.
AWS Credential expired, need to generate new credentials
Assume Role MFA token code: 845964
refreshed AWS credentials
Found 1 ASG groups for deletion
Deleted ASG group cluster-ulli10-tcnp-vu93au66da-NodePoolAutoScalingGroup-KNdNlJ0pRcx2
Terminated 3 instances in ASG group cluster-ulli10-tcnp-vu93au66da-NodePoolAutoScalingGroup-KNdNlJ0pRcx2
Deleted vintage vu93au66da node pool ASG.

Executing the following command to apply non-default apps to CAPI MC via external tool:
app-migration-cli apply -s garfish -d grizzly -n ulli10 -o org-ulli
Connected to gs-garfish, k8s server version v1.24.17
Connected to gs-grizzly, k8s server version v1.25.16

All prerequistes are found on the new MC for app migration
Applying all non-default APP CRs to MC
All non-default apps applied successfully.

Apps (0) applied successfully to grizzly-ulli10
Finished migrating cluster ulli10 to CAPI infrastructure
=======================================
Cluster was successfully migrated to CAPI
Now you can check the cluster and confirm  everything was properly migrated
```

**Comment by alex-dabija** [comment]
> All apps are migrated and running on/from the capi mc. Some minor issues found on the helm labels, which i'll think about.

@nce How critical are these issues? Can we move forward with migrating customer clusters? Are there followup issues created?

**Comment by nce** [comment]
I have found them non-blocking. i think it was an issue created from apps which i was testing in an "out of our scope"-setting (apps belonging on an MC, but deployed to a WC). 

We can definitely move forward
